core/templates/dev/head/components/forms/html_select_directive.html

```diff
@@ -1,12 +1,11 @@
 <div uib-dropdown class="oppia-html-select">
   <button uib-dropdown-toggle class="btn btn-default oppia-html-select-selection" type="button" aria-expanded="false" data-toggle="dropdown">
     <div>
-      <angular-html-bind class="oppia-html-select-selection-element" html-data="options[getSelectionIndex()].val">
-      </angular-html-bind>
+      Click here to select
     </div>
     <span class="caret oppia-html-select-selection-caret"></span>
   </button>
-  <ul uib-dropdown-menu style="width: inherit;">
+  <ul uib-dropdown-menu style="max-width: 1000px; width: auto; overflow: auto;">
```

Comment:
Why the `max-width: 1000px` ?

Reply:
@bansalnitish I made a change. It looks pretty fine now.
![image](https://user-images.githubusercontent.com/32777813/50049739-ee900b00-0111-11e9-92f4-109d95ecc7af.png)
Have a look.
Thanks.